umm-spec-lib/test/cmr/umm_spec/test/umm_g/granule.clj

```diff
@@ -25,7 +25,15 @@
           actual (core/parse-metadata {} :granule :umm-json umm-g-metadata)
           expected (expected-util/umm->expected-parsed granule)
           actual (update-in actual [:spatial-coverage :geometries] set)
-          expected (update-in expected [:spatial-coverage :geometries] set)]
+          expected (update-in expected [:spatial-coverage :geometries] set)
```

Comment:
Can we pull all this extra processing into its own function we can add to, to keep the test clean and readable.

Reply:
the set logic could also be included.